### ISSUE TITLE
Doesn't handle multiple admin interface declarations.

### ISSUE THREAD

**Issue by daveallen** [ernestas-poskus/chef-proxysql#29]
Me again.  lol

I have had an issue when I set up multiple admin interfaces in the admin_variables section:
  mysql_ifaces: '127.0.0.1:6032;/var/run/proxysql/proxysql.sock'

The code looks to see if there is a .sock in the string and then adds the string to the command that is used in the load-config command.

        connection = if admin_mysql_ifaces =~ /\.sock/
                       "--socket #{admin_mysql_ifaces}"
                     else
                       host, port = admin_mysql_ifaces.split(':')
                       "--host #{host} --port #{port}"
                     end

And I get this error:
mysql --user=admin --password=admin --socket localhost:6032;/var/run/proxysql/proxysql.sock returned 2

I have fixed it with this code:
        connection = if admin_mysql_ifaces =~ /\.sock/
                       iface = admin_mysql_ifaces.split(";").select {|inf| inf =~ /\.sock/}
                       "--socket #{iface.first}"
                     else
                       host, port = admin_mysql_ifaces.split(':')
                       "--host #{host} --port #{port}"
                     end

Thanks again for this cookbook!

**Comment by ernestas-poskus** [comment]
Ahh yes I see, @daveallen maybe you would like to contribute a pull request with this fix :) ?

**Comment by daveallen** [comment]
Yes, I will give you a PR.

**Comment by daveallen** [comment]
Here it is.  https://github.com/ernestas-poskus/chef-proxysql/pull/32

**Comment by ernestas-poskus** [comment]
Released 2.3.0 https://supermarket.chef.io/cookbooks/proxysql

Thank you

**Comment by petracvv** [comment]
Hey, can this be reopened? I am seeing a similar problem but using two IP addresses in the mysql_ifaces variable:
```
mysql_ifaces: '127.0.0.1:6032;10.10.10.10:6032'
```

Will produce a string like: 
```
mysql --user="admin" --password="admin" --host 127.0.0.1 --port 6032;10.10.10.10
```

The code doesn't take into affect multiple interface declarations (that are not sockets)
https://github.com/vinted/chef-proxysql/blob/6b1150547c53e81d02c44964c459d49e8cef1ba8/libraries/proxysql_service.rb#L190-L193

At the moment I can work around this by adding an extra socket declaration to the mysql_ifaces variable

**Comment by ernestas-poskus** [comment]
Published fix in 3.4.1 https://supermarket.chef.io/cookbooks/proxysql
